### PR TITLE
feat: add `bytes_sent` and `bytes_received` as metrics

### PR DESCRIPTION
**Pull request by jackwotherspoon**
Adding additional metrics to be reported by Go Connector.

The Connector will now report the number of `bytes_sent` to Cloud SQL
and `bytes_received` from Cloud SQL.

### CODE REVIEW

File: internal/trace/metrics.go [hunk 1]
```diff
@@ -94,6 +104,20 @@ var (
 		Aggregation: view.Count(),
 		TagKeys:     []tag.Key{keyInstance, keyDialerID, keyErrorCode},
 	}
+	bytesSentView = &view.View{
+		Name:        "cloudsqlconn//bytes_sent",
```
Comment (enocom) [inline internal/trace/metrics.go:108]: Double slash here.

Comment (enocom) [reply internal/trace/metrics.go:108]: Ditto below.

**Review by enocom — COMMENTED** [review]
Shall we add a test in https://github.com/GoogleCloudPlatform/cloud-sql-go-connector/blob/main/metrics_test.go for this?

**Comment by jackwotherspoon** [comment]
Screenshot of metrics appearing in Cloud Monitoring:

`bytes_sent`:
image

`bytes_received`:
image

**Review by enocom — APPROVED** [review]

**Pull request merged**